### PR TITLE
Add screenshots for release 2.9

### PR DESCRIPTION
**Pull request by larswmh**
This pull request adds the screenshot for the soon to be released new 2.9 version of the Corona-Warn-App. It's not a pull request to the master branch, but to a release 2.9 branch that will get merged with every update when the new release is out

### CODE REVIEW

**Comment by Ein-Tim** [comment]
Issues with this PR (some issues might be already reported as a separate issue, these I did not write down again):

### iOS 

- "In Vertretung warnen"
	- "In Vertretung warnen Bild 5 von 6" keyboard is broken

- "Representative warning" 
	- Rename to "Warn for others"


Btw @dsarkar, what do you think, should we create a label "Screenshots Issue" so that we keep an overview over open issues reg. the screenshots page (also in the Android repo)? Currently, I have to admit, I find it quite hard to keep an overview over open issues reg. the screenshots 😅

File: src/data/screenshots_de.json [hunk 1]
```diff
@@ -5,14 +5,16 @@
...
-                "Neue Funktionen in Version 2.8 sind die Standardisierung der Namen und die Möglichkeit, Zertifikate aus Testergebnissen zu öffnen.",
-                "Neben dem Namen in der länderspezifischen Schreibweise wird nun in jedem Zertifikat auch der Name in standardisierter, lateinischer Schreibweise angezeigt.",
-                "Sie können nun in der App direkt aus dem Testergebnis Ihr digitales COVID-Testzertifikat der EU abrufen, sofern ein Zertifikat verfügbar ist. Diese Funktion ist bisher nur für Android verfügbar, wird aber zukünftig auch für iOS verfügbar sein.",
+                "Verschaffen Sie sich mithilfe von Screenshots einen Überblick über die Funktionen, die die Corona-Warn-App mit Version 2.9 (verfügbar seit 08.09.2021) bietet. Bitte beachten Sie, dass die Screenshots für Android und iOS designbedingt leicht voneinander abweichen können.",
+                "Neue Funktionen in Version 2.9 sind das Warnen in Vertretung, die sofortige Impfstatusanzeige für Genesene, der Aufruf von digitalen COVID-Testzertifikaten der EU aus einem Testergebnis (zuvor nur auf Android möglich) und das Hinzufügen einer Auffrischimpfung.",
```
Comment (Ein-Tim) [inline src/data/screenshots_de.json:13]: In the general public the term "Auffrischungsimpfung" is used, the BZGA uses it: https://www.infektionsschutz.de/coronavirus/schutzimpfung/fragen-und-antworten/auffrischungsimpfung.html.
So maybe we should replace all occurrences of "Auffrischimpfung" with "Auffrischungsimpfung"?

Comment (larswmh) [reply src/data/screenshots_de.json:13]: Changed in next commit, thanks

**Comment by larswmh** [comment]
> Issues with this PR (some issues might be already reported as a separate issue, these I did not write down again):
> 
> ### iOS
> * "In Vertretung warnen"
>   
>   * "In Vertretung warnen Bild 5 von 6" keyboard is broken
> * "Representative warning"
>   
>   * Rename to "Warn for others"
> 
> Btw @dsarkar, what do you think, should we create a label "Screenshots Issue" so that we keep an overview over open issues reg. the screenshots page (also in the Android repo)? Currently, I have to admit, I find it quite hard to keep an overview over open issues reg. the screenshots 😅

Fixed the broken keyboard and changed the category naming to "Warn for others", since that is how it is mentioned in the app. Thanks! @Ein-Tim

**Comment by MikeMcC399** [comment]
@larswmh 
Please see issue https://github.com/corona-warn-app/cwa-website/issues/1710 "Screenshots: Valid certificates in 'Invalid certificates' section (Android)" which affects screenshots for version 2.8 and screenshots in this PR for 2.9.

**Comment by MikeMcC399** [comment]
@larswmh 
When viewing this branch locally, I'm seeing iOS vaccination certificates expired
http://localhost:8000/en/screenshots/#ios_vaccination_certificate
I don't think that this should be the case. It's the same issue as in the published 2.8 screenshots
https://www.coronawarn.app/en/screenshots/#ios_vaccination_certificate

**Comment by MikeMcC399** [comment]
@larswmh 
PR https://github.com/corona-warn-app/cwa-website/pull/1711 regarding Android Invalid certificates needs to be applied to this branch as well. What is the best way to do this?

**Comment by dsarkar** [comment]
@Ein-Tim 

> Btw @dsarkar, what do you think, should we create a label "Screenshots Issue" so that we keep an overview over open issues reg. the screenshots page (also in the Android repo)? Currently, I have to admit, I find it quite hard to keep an overview over open issues reg. the screenshots 😅

Agreed!

**Comment by larswmh** [comment]
@MikeMcC399 valid screenshots in invalid category have been changed to invalid screenshots. For the other issue about expired vaccination certificates in the vaccination certificates category, we don't have any issues as of now

**Comment by MikeMcC399** [comment]
@larswmh 
> valid screenshots in invalid category have been changed to invalid screenshots. 

Thank you! I have closed issue https://github.com/corona-warn-app/cwa-website/issues/1710.

> For the other issue about expired vaccination certificates in the vaccination certificates category, we don't have any issues as of now

Are you saying that it is expected to show expired vaccination certificates for iOS in 

https://www.coronawarn.app/en/screenshots/#ios_vaccination_certificate (for 2.8)
en/screenshots/#ios_vaccination_certificate (for 2.9 in npm start localhost)

image

**Comment by larswmh** [comment]
@MikeMcC399 glad you've mentioned it again. I didn't mean to say we don't have any issues as of now, I meant to say we **don't have any assets** as of now. For example, valid examples for test and recovery certificates exist but not for vaccination certificates.

**Review by dsarkar — APPROVED** [review]

**Pull request merged**